packages/web3torrent/src/utils/torrent-status-checker.ts

```diff
@@ -76,9 +76,6 @@ export function torrentStatusChecker(
       length: live.length || previousData.length,
       downloaded: live.downloaded || 0,
       status: getStatus(live, web3Torrent.pseAccount),
-      uploadSpeed: live.uploadSpeed,
```

Comment:
I think these 3 lines are making problems:
![image](https://user-images.githubusercontent.com/10502605/79395209-893f6a80-7f4f-11ea-944f-47b3b0b7d23f.png)

Reply:
The number of peers and the download/upload speed it's not showing on the leecher nor the seeder.

Reply:
Good catch! Do you know if this an issue on the current `master`, or just on this branch?

Reply:
Regarding the lines this comment is referencing: these lines are deleted because the lines will never have any effect.

Reply:
> Good catch! Do you know if this an issue on the current master, or just on this branch?

Just this branch, I think

Reply:
I believe this should be fixed in a4a137545.

Reply:
<img width="931" alt="Screen Shot 2020-04-16 at 1 55 26 PM" src="https://user-images.githubusercontent.com/1062379/79506104-467ea080-7fea-11ea-9581-6eb2819978f7.png">

Reply:
awesome!